test/Test_dynamics.cpp

```diff
@@ -75,7 +104,7 @@ TEST_CASE("Dynamics") {
         dynamics.addAgent(0, 2);
         THEN("The agent is added") {
           CHECK_EQ(dynamics.agents().size(), 1);
-          const auto& agent = dynamics.agents().at(0);
+          const auto &agent = dynamics.agents().at(0);
```

Comment:
Have you changed something of `clang-format`?

Reply:
I'm guessing that the new clang version on fedora changed a few things